contrib/internal/httptrace/httptrace.go

```diff
@@ -50,9 +51,23 @@ func StartRequestSpan(r *http.Request, opts ...ddtrace.StartSpanOption) (tracer.
 			if r.Host != "" {
 				cfg.Tags["http.host"] = r.Host
 			}
-			if spanctx, err := tracer.Extract(tracer.HTTPHeadersCarrier(r.Header)); err == nil {
+
+			spanctx, err := tracer.Extract(tracer.HTTPHeadersCarrier(r.Header))
+			is_inferred_proxy_set := false
+			println("IN HERE inferredProxyEnabled is:")
```

Comment:
Just want to make sure we get rid of these debug print statements before approving